src/Microsoft.AspNet.Mvc.Core/ViewContext.cs

```diff
@@ -19,6 +19,16 @@ public class ViewContext : ActionContext
         private DynamicViewData _viewBag;
 
         /// <summary>
+        /// Creates a empty <see cref="ViewContext"/>.
```

Comment:
Looks like the same typo is here: https://github.com/aspnet/Mvc/blob/dev/src/Microsoft.AspNet.Mvc.Core/ActionContext.cs#L17